src/genomic_features/_core/filters.py

```diff
@@ -211,6 +220,99 @@ def required_tables(self) -> set[str]:
         return {"gene"}
 
 
+class SeqFilter(AbstractFilterEqualityExpr):
+    """Filter by seq_name (e.g. chromosome).
+
+    Usage
+    -----
+
+    >>> ensdb.genes(filter=gf.filters.SeqFilter("MT"))
+    """
+
+    def columns(self) -> set[str]:
+        return {"seq_name"}
+
+    def required_tables(self) -> set[str]:
+        return {"gene"}
+
+
+class CanonicalFilter(AbstractFilterExpr):
```

Comment:
maybe I would call this `CanonicalTranscriptFilter` instead

Reply:
Agreed!